packml_msgs/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package>
+  <name>packml_msgs</name>
+  <version>0.0.0</version>
+  <description>Packml messages</description>
+  <maintainer email="shaun.edwards@gmail.com">Shaun Edwards</maintainer>
+  <license>Apache 2.0</license>
+  <author >Shaun Edwards</author>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>message_generation</build_depend>
+  <build_depend>std_msgs</build_depend>
+
+  <run_depend>message_runtime</run_depend>
+  <run_depend>std_msgs</run_depend>
+
+</package>
```

Comment:
re: generate version 2: no, not yet (there is a PR against catkin for that).

But I would really recommend to just use v2 for all new packages.